Standard/BisCustomAttributes.ecschema.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="BisCustomAttributes" alias="bisCA" version="01.00.00" description="Custom attributes to indicate BIS concepts." displayLabel="BIS Custom Attributes" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECEnumeration typeName="SchemaGradeValue" backingTypeName="string" isStrict="true" description="Used with SchemaType to declare the schema author's intended grade for the schema.">
+        <ECEnumerator displayLabel="A" value="A"/> <!-- description="True BIS schemas carefully designed for editing and interoperability." -->
```

Comment:
EC 3.2 added support for descriptions in ECEnumerators:
Spec doc:
https://www.itwinjs.org/bis/ec/ec-enumeration/#sub-elements
EC 3.2 xsd
https://github.com/iTwin/bis-schemas/blob/6060565c0db3a596c09b9c0b00addac1bbdaf32f/Standard/xsd/ECSchemaXML3.2.xsd#L336
EC 3.1 xsd
https://github.com/iTwin/bis-schemas/blob/6060565c0db3a596c09b9c0b00addac1bbdaf32f/Standard/xsd/ECSchemaXML3.1.xsd#L425

Reply:
I followed the naming convention used by the ProductionStatus CA, but I like your naming convention better. I changed the schema and remarks accordingly.